css-canvas-medical/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header-container">
+    <div class="header-row-1 padding-header-row-1">
+      <div class="header-col-1 display-header-col-1">
+        <img src="images/logo-blue.svg" alt="canvas-logo-blue">
+      </div>
+      <div class="header-col-2 display-header-col-2">
+        <div class="header-col-2-text">
+          <p>Product</p>
+        </div>
+        <div class="header-col-2-text">
+          <p>Implementation</p>
+        </div>
+        <div class="header-col-2-text">
+          <p>About&nbsp;Us</p>
+        </div>
+        <div class="header-col-2-text">
+          <p>Blog</p>
+        </div>
+      </div>
+      <div class="header-col-3">
+        <button>Contact Us</button>
+      </div>
+    </div>
```

Comment:
Layout classes should not be named after specific sections. You should have one `container` class, one `row` class that you can apply to any row in the layout, and one variant of `column-` classes for each size, that is reused for each element that needs to be that size.